it/test/uk/gov/hmrc/tradergoodsprofiles/controllers/MaintainProfileControllerIntegrationSpec.scala

```diff
@@ -193,140 +196,15 @@ class MaintainProfileControllerIntegrationSpec
       )
     }
 
-    "return 400 Bad Request when actorId is missing from body or is in the wrong format" in {
-      withAuthorizedTrader()
```

Comment:
Similarly, I believe this we should. keep this test case in integration test. It verifies the expected error response from router